backend.py

```diff
@@ -265,7 +265,37 @@ def create_result(self, std_out, std_err, status="intermediate", files=False):
             for name in filenames:
                 # ToDO: if filesize > 1mb -> generate s3-url
                 file_path = os.path.join(self.tmp_dir, "files", name)
-                mimetype = mime.from_file(file_path)
+                mimetypes.init()
+                mimetypes.add_type("application/vnd.kitware", ".vtu")
+                mimetypes.add_type("application/vnd.kitware", ".vtp")
+                mimetypes.add_type("application/x-vgf", ".vgf")
+                mimetypes.add_type("application/x-vgf3", ".vgf3")
+                mimetypes.add_type("application/x-vgfc", ".vgfc")
+                mime = mimetypes.guess_type(file_path)
+                mimetype = "" 
+                if mime[0] == "text/plain":
+                    fr = open(file_path, "r")
+                    file = fr.readlines()
```

Comment:
use "with", see lines 299ff for an example;
don't use varibale name "file" - it is a reserved keyword in python